src/register/RegistrationPage.jsx

```diff
@@ -227,10 +227,8 @@ const RegistrationPage = (props) => {
   const handleOnChange = (event) => {
     const { name } = event.target;
     const value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
-    if (registrationError[name]) {
-      dispatch(clearRegistrationBackendError(name));
-      setErrors(prevErrors => ({ ...prevErrors, [name]: '' }));
-    }
+    dispatch(clearRegistrationBackendError(name));
```

Comment:
i noticed that, by this change, redux action is being called on every character press. Which is performance degrade, so we will have to check this as well.

Reply:
[For my understanding] what is this change doing?

Reply:
Has this been addressed, yet?  We should probably be debouncing this.

Reply:
@arbrandes thank you! IDK how it was missed. But yes, this has been addressed now by moving `dispatch(clearRegistrationBackendError(name));` back inside the `if (registrationError[name])` condition. 

This has resolved both the original bug and also prevented calling the Redux action on each character press.

@DmytroAlipov FYI^